module/VuFind/src/VuFind/View/Helper/Root/Matomo.php

```diff
@@ -418,7 +420,9 @@ protected function getRecordPageCustomData(RecordDriverBase $recordDriver): arra
      */
     protected function getLightboxCustomData(): array
     {
-        return [];
+        return [
+            'Context' => $this->context
```

Comment:
Is there a reason this is defaulted to page everywhere but here? (I assume there is, but maybe it's worth adding a comment since this is an exception to a pattern seen elsewhere in the code).

Reply:
getLightboxCustomData should never be called in normal page context. There's no harm in making it identical to the others, though, so I'll just change it.

Reply:
Changed.